src/cgame/cg_event.cpp

```diff
@@ -212,26 +212,47 @@ static void CG_Obituary( entityState_t *ent )
 			{
 				if ( meansOfDeath[ mod ].showAssist && assistantInfo )
 				{
-					Log::Notice( "%s (+ %s%s^*) %s %s%s", teamTag[ attackerTeam ], teamTag[ assistantTeam ], assistantName, meansOfDeath[ mod ].icon, teamTag[ ci->team ], targetName );
+					Log::Notice( "%s (+ %s%s%s^*) %s %s%s", 
+					             teamTag[ attackerTeam ], 
+								 teamTag[ assistantTeam ], 
+								 assistantName, 
+								 ( assistantTeam == ci->team ? " ^a!!" : "" ),
+								 meansOfDeath[ mod ].icon, 
+								 teamTag[ ci->team ], targetName );
 				}
 				else
 				{
-					Log::Notice( "%s %s %s%s", teamTag[ attackerTeam ], meansOfDeath[ mod ].icon, teamTag[ ci->team ], targetName );
+					Log::Notice( "%s %s %s%s", 
+					             teamTag[ attackerTeam ], 
+								 meansOfDeath[ mod ].icon, 
+								 teamTag[ ci->team ], targetName );
 				}
 			}
 			else if ( attacker == target )
 			{
-				Log::Notice( "%s %s%s", meansOfDeath[ mod ].icon, teamTag[ ci->team ], targetName );
+				Log::Notice( "%s %s%s", 
+				             meansOfDeath[ mod ].icon, 
+							 teamTag[ ci->team ], targetName );
 			}
 			else
 			{
 				if ( meansOfDeath[ mod ].showAssist && assistantInfo )
 				{
-					Log::Notice( "%s%s^* (+ %s%s^*) %s %s%s", teamTag[ attackerTeam ], attackerName, teamTag[ assistantTeam ], assistantName, meansOfDeath[ mod ].icon, teamTag[ ci->team ], targetName );
+					Log::Notice( "%s%s%s^* (+ %s%s%s^*) %s %s%s", 
+					             teamTag[ attackerTeam ], attackerName, 
+								 ( attackerTeam == ci->team ? " ^1!!!^*" : "" ),
+								 teamTag[ assistantTeam ], assistantName, 
+								 ( assistantTeam == ci->team ? " ^a!!" : "" ),
```

Comment:
They're a different colour for assist teamkills.

Reply:
> How do you use emoticon kill messages?

For what it's worth, I never answered this. Most people won't even know they exist because I don't see an option to enable it. The cvar is `cg_emoticonsInMessages`. I only discovered it when looking to fix another obituary bug yesterday...

There should probably be a PR for this to add the option to the menu.

Reply:
We can do better in the future if we think we can do better. This is only a detail anyway this should not block the merge.